### PR TITLE
UIREQ-941 JEST/RTL test cases for SortableList

### PR DESCRIPTION
**Pull request by KetineniM**
UIREQ-941 JEST/RTL test cases for SortableList
URL: https://issues.folio.org/browse/UIREQ-941

### CODE REVIEW

**Review by cchitneedi — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
 * Added requestDate token. Refs UIREQ-962.
 * Update `request-storage` okapi interface to `6.0` version. Refs UIREQ-963.
 * UI tests replacement with RTL/Jest for src/PatronBlockModal.js. Refs UIREQ-878.
+* create Jest/RTL test for SortableList.js. Refs UIREQ-941.
```
Comment (artem-blazhko) [inline CHANGELOG.md:27]:
```suggestion
* Create Jest/RTL test for SortableList.js. Refs UIREQ-941.
```

File: src/components/SortableList/SortableList.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+const renderSortableList = (prop) => {
+  const Component = () => SortableList(prop);
+  return (
+    render(<Component />)
+  );
+};
+
+describe('renderSortableList', () => {
```
Comment (artem-blazhko) [inline src/components/SortableList/SortableList.test.js:25]:
```suggestion
describe('SortableList', () => {
```

**Review by artem-blazhko — CHANGES_REQUESTED** [review]
Could you check that MultiColumnList and `provided.placeholder` were rendered?

**Review by ssandupatla — APPROVED** [review]

File: src/components/SortableList/SortableList.test.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    expect(document.querySelector('[droppableid="droppableId1"]')).toBeInTheDocument();
+  });
+  it('Should render the MultiColumnList', () => {
+    renderSortableList(propsData);
+    expect(screen.getByText('MultiColumnList')).toBeInTheDocument();
+  });
+  it('Should render the Provided Placeholder', () => {
+    renderSortableList(propsData);
+    expect(screen.getByText('Placeholder')).toBeInTheDocument();
+  });
```
Comment (artem-blazhko) [inline src/components/SortableList/SortableList.test.js:42]:
```suggestion
  beforeEach(() => {
    renderSortableList(propsData);
  });

  it('Should render the SortableList component with provided droppableId', () => {
    expect(document.querySelector('[droppableid="droppableId1"]')).toBeInTheDocument();
  });
  
  it('Should render the MultiColumnList', () => {
    expect(screen.getByText('MultiColumnList')).toBeInTheDocument();
  });
  
  it('Should render the Provided Placeholder', () => {
    expect(screen.getByText('Placeholder')).toBeInTheDocument();
  });
```

**Review by artem-blazhko — APPROVED** [review]
Could you change my single comment and then we can merge it?

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**